src/main/java/fr/techad/sonar/gerrit/ReviewInput.java

```diff
@@ -22,12 +22,16 @@
     private Map<String, Integer> labels = new ConcurrentHashMap<String, Integer>();
     private Map<String, List<ReviewFileComment>> comments = new ConcurrentHashMap<String, List<ReviewFileComment>>();
 
+    public void setValueAndLabel(@NotNull String value, @NotNull String label) {
```

Comment:
Only integer value are accepted. Why do you not use int directly ?
GerritConfiguration could return directly the int.